package.json

```diff
@@ -61,5 +61,8 @@
     "tabWidth": 2,
     "semi": true,
     "singleQuote": false
+  },
+  "dependencies": {
+    "isomorphic-fetch": "^3.0.0"
   }
```

Comment:
Remove this, this should be only on lib/package.json

Reply:
What would be the `isomorphic-fetch` used for?

Reply:
@alexjoverm Removed✅ @fgiuliani We implemented support for the use case without Fetch API support as we imagined such requests would come from users.